.github/workflows/test.yml

```diff
@@ -12,17 +12,17 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        environment: [ centos-7, centos-8, debian-10 ]
+        environment: [ centos-7, aws-centos-8, debian-10 ]
```

Comment:
It would increase readability if the order in the environment array would be the same as in the include array.

Reply:
Right. While I was at it, I renamed the older tags to be more meaningful in terms of the used cloud provider. Thx